src/wesnoth_lua_config.md

```diff
@@ -4,7 +4,7 @@ This document describes the process used to install Lua 5.4.4 for Wesnoth. The e
 
 The goal of this process was, as much as possible, install clean, unchanged sources.
 Traditionally, Wesnoth maintainers have made changes directly to the Lua source kit.
-__This is strongly discouraged.__
+__This is no longer allowed.__
```

Comment:
Adding ", because it will break ..." would be good here, otherwise it's just waiting for someone to say "but I need to to do X, thus this rule is discarded".

Reply:
Good idea, thanks.  Done, with other similar documentation improvements.